features/providers/step_definitions/civil_journey_steps.rb

```diff
@@ -118,13 +122,13 @@
     applicant: applicant
   )
   proceeding_type = create(:proceeding_type)
-  legal_aid_application = create(
+  @legal_aid_application = create(
```

Comment:
Why does this have to be an instance variable? Is it used outside the scope of the current context? If not a normal variable (`legal_aid_application`) is better as it limits the scope of the variable to just the `Given('I complete the journey as far as check your answers')` context

Reply:
it's used in the `Given('I used delegated functions') do` block on line 103

Reply:
OK

Reply:
FYI, Rob decided that he will create a new PR to add the updated cucumber test later because he is the one who worked on this part of the flow